dc_config/images/celery/requirements.txt

```diff
@@ -1,3 +1,2 @@
 celery==5.1.2
-pymongo==3.12.0
-
+pymongo==3.7.2
```

Comment:
Did the newer version of pymongo throw any errros?

Reply:
I updated when changing the insert_many and Insert_one code. No errors on my side. I needed to upgrade to 3.2 but thought why not go to the current stable version.

Reply:
I will test and confirm on my end. I agree to use the latest version if possible.